### ISSUE TITLE
v0.21.0 can't connect to RDS db when running on AWS ECS

### ISSUE THREAD

**Issue by dmonaco05** [camptocamp/terraboard#118]
background:
Running postgres on an RDS instance
Trying to run terraboard on ECS (EC2 instance)

When I try to run terraboard's docker image on AWS ECS I get an error on startup that the RDS DB instance is busy
```
level=info msg="Terraboard v0.21.0 (built for Terraform v0.12.28) is starting..."
level=info msg="Using AWS (S3+DynamoDB) as the state/locks provider"
level=fatal msg="dial tcp: lookup terraboard.abcd1234.us-east-1.rds.amazonaws.com: device or resource busy"
```

I confirmed all my configs/permissions are correct by running the container locally with no issues, also confirmed that the ECS instance the container runs on is able to connect to the RDS instance directly so that end is setup correctly as well.

**Comment by raphink** [comment]
This looks to me like a DNS issue, maybe due to the statically built binary.

**Comment by dmonaco05** [comment]
> This looks to me like a DNS issue, maybe due to the statically built binary.

What statically built binary?
Not sure how that could be a DNS issue, I already stated that the instance the container runs on is able to connect to RDS.

**Comment by raphink** [comment]
How did you run Terraboard? From the provided binary, from the provided Docker image, or did you build it yourself?

**Comment by dmonaco05** [comment]
docker image v0.21.0

**Comment by raphink** [comment]
Then you're using a static binary, since Terraboard is built statically in the Docker image, in order to use a `scratch` image as template.

Several go binaries have reports of DNS request failures when built statically, though usually this seems to happen only with names ending in `.local` (see https://github.com/containers/skopeo/issues/636 for ex)

I wonder if you're hitting a similar bug here.

**Comment by mihaiplesa** [comment]
Just pass `GODEBUG: netdns=go` to the Terraboard container as per https://golang.org/pkg/net/#hdr-Name_Resolution

**Comment by raphink** [comment]
Should be fixed in #135
